src/components/TextField/README.md

```diff
@@ -872,6 +872,7 @@ Text fields have standard keyboard support.
 - If the `type` is set to `number`, then merchants can use the up and down arrow keys to adjust the value typed into the field
 - Using the `disabled` prop will prevent the text field from receive keyboard focus or inputs
 - The `readOnly` prop allows focus on the text field but prevents input or editing
+- The `inputMode` prop can be used to bring up a relevant keyboard for merchants on mobile
```

Comment:
On a somewhat related topic… I see there’s a `KeyboardAccessories` component here: “https://polaris.shopify.com/components/forms/keyboard-accessories#section-best-practices”, but there’s only documentation in the repo? Am confused.

Reply:
I feel like this should more explicitly state that it is just passing the property though, and maybe link to the doc about this property on [developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode) for example.

Reply:
Didn’t see this done for the other attributes but yeah, makes total sense, I’ll do that

Reply:
Done!